prbt_default_application/launch/my_application.launch

```diff
@@ -36,19 +39,24 @@ limitations under the License.
     <arg name="fake_execution" value="$(arg sim)"/>
     <arg name="pipeline" value="pilz_command_planner" /><!-- Choose planning pipeline -->
     <arg name="load_robot_description" value="false" />
+    <arg name="gripper" value="$(arg gripper)" />
     <!-- <arg name="capabilities" value="" /> --> <!-- additional capabilites (space seperated) -->
     <!-- <arg name="disable_capabilities" value="" /> --> <!-- inhibit capabilites (space seperated) -->
   </include>
 
-  <!-- The semantic description that corresponds to the URDF -->
+  <!-- The semantic description that corresponds to the URDF.
+       Options: default srdf / custom file -->
   <param name="robot_description_semantic" command="$(find xacro)/xacro --inorder
-               $(find prbt_default_application)/config/prbt.srdf.xacro" />
+               $(find prbt_moveit_config)/config/prbt.srdf.xacro gripper:=$(arg gripper)" >
```

Comment:
Did you switch to prbt_moveit_config on purpose?

Reply:
I think I said it again, but it would be better to not have launch files in our packages (only in these templates)
And especially to not include them here. This encourages the user to change something in the included launchfile instead of creating it himself.

Reply:
@rfeistenauer Yes, to allow the user copy-n-paste only the launch file without a whole package of includes. The custom file is still there (commented out). What would be a better comment to describe the users' options?

Reply:
@ct2034 I think that we should continue installing the `prbt_moveit_config` package. Other robots provide this convenience package as well. I like the installation approach as quick-start

```
sudo apt install pilz_robots
roslaunch prbt_moveit_config moveit_planning_execution.launch
```

Can you make a proposal, where the user can startup the pilz robot packages with two or three command lines with your approach? Creating and building a custom workspace is a whole ROS beginners tutorial, that would be required as prerequisite...

Reply:
You are right, must have missed that line. Gues it's ok tat way

Reply:
@jschleicher Maybe you are right.